src/cgtop/cgtop.c

```diff
@@ -672,7 +673,7 @@ static void help(void) {
                "  -n --iterations=N   Run for N iterations before exiting\n"
                "  -b --batch          Run in batch mode, accepting no input\n"
                "     --depth=DEPTH    Maximum traversal depth (default: %u)\n"
-               "  -M --machine=       Show container\n"
+               "  -M --machine=       Show container (incompatible when cgroup path is specified)\n"
```

Comment:
uh, oh, I am not sure either way. I can see @zonque's point, but I also see the benefit to making this similar to cgls... That said, there's of course a discrepancy here, given that cgls handles multiple cgroup paths fine, and this tool does not... hmm, but i figure being like cgls is probably the better option, as long as there's a nice error message thrown...